modules/mobile-agents/android/jax-rs/src/main/webapp/WEB-INF/cxf-servlet.xml

```diff
@@ -23,7 +23,7 @@
          http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
          http://cxf.apache.org/jaxrs http://cxf.apache.org/schemas/jaxrs.xsd">
 
-    <jaxrs:server id="operationService" address="/operations">
+    <jaxrs:server id="operationService" address="/operation">
```

Comment:
The reason that I've changed here is to maintain the consistency across all the JAX-RS implementation. On the other hand this endpoint is rather mapped to single resource.
For ex: /operation/lock, /operation/mute likewise.